data_request_api/stable/content/dump_transformation.py

```diff
@@ -526,11 +575,11 @@ def get_transformed_content(version="latest_stable", export_version="release", u
             if os.path.exists(VS_content):
                 os.remove(VS_content)
         if not(all(os.path.exists(filepath) for filepath in [DR_content, VS_content])):
-            content = dc.load(version, export=export_version, consolidate=use_consolidation)
+            content = dc.load(version, export=export, consolidate=consolidate)
             data_request, vocabulary_server = transform_content(content, version)
             write_json_output_file_content(DR_content, data_request)
             write_json_output_file_content(VS_content, vocabulary_server)
-        return DR_content, VS_content
+        return dict(DR_input=DR_content, VS_input=VS_content)
 
 
 if __name__ == "__main__":
```

Comment:
The VS and DR dictionaries could be written per default to `dc._dreq_res + version`  so the files would be cached/located alongside the release and raw export files?